modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/ListItem.js

```diff
@@ -0,0 +1,6 @@
+import ListItem from '@tiptap/extension-list-item';
+export default (options) => {
+  return ListItem.extend({
+    content: 'defaultNode block*'
```

Comment:
looking at the original extension, so we *never* encounter `paragraph` the way this would expect because we always replace it, is that correct? is there any reason to add on to the default ie `defaultNode paragraph block*` ?